packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -338,7 +338,67 @@ export async function handleConfirmation(args: {
         );
       })
     );
+
     await Promise.all(promises);
+
+    if (paid) {
```

Comment:
Added a whole section when receiving paid flag here

Reply:
How about refactoring it out to a `handlePaidEventConfirmation` function